openapi/openapi.yaml

```
@@ -192,6 +192,7 @@ tags:
       - `restrict-to-plans`: Restricts a coupon to specific pricing plans.
       - `restrict-to-subscriptions`: Restricts a coupon to specific order subscriptions.
       - `restrict-to-customers`: Restricts a coupon to specific order subscriptions.
+      - `restrict-to-exclusive-appliance`: Determines whether a coupon may be used in combination with other coupons. A coupon with this restriction enabled is applied only if it gives a bigger discount than the other (non-exclusive) coupon(s) together. If those conditions are met and this coupon is applied, all other coupons are removed.
```

suggestion: for readability
(based on my other suggestions)
```suggestion
      - `restrict-to-exclusive-appliance`: Restricts a coupon so that it cannot be used in combination with other coupons. If more than one coupon is active, a coupon with this restriction is only applied if it provides a larger discount than the other coupons combined. If a coupon with this restriction is applied, all other coupons are removed.
```